language/tools/move-mv-llvm-compiler/src/main.rs

```diff
@@ -140,12 +165,13 @@ fn main() -> anyhow::Result<()> {
             tgt_platform.llvm_cpu(),
             tgt_platform.llvm_features(),
         );
-        let mod_id = model_env
+        let args = Args::parse();
```

Comment:
This second call to `Args::parse` is surprising Can it be avoided and the original args reused?

Reply:
The original `args` were borrowed at this point.
You may suggest a better way but re-parsing looked very simple for me.